src/client/css/global.css

```diff
@@ -189,23 +187,58 @@ button.primary,
   transition: background 150ms;
 }
 
+button.secondary,
+.button.secondary {
+  color: var(--blue-50);
+  border: 1px solid;
+  padding: var(--padding-md);
+}
+
+button.primary:active,
+.button.primary:active,
+button.secondary:active,
+.button.secondary:active {
+  box-shadow: inset 0 0 64px #80808011;
```

Comment:
Question (non-blocking): Should we create CSS variables for shadows so we can try to reuse them as much as possible instead of creating new ones in the future?

Reply:
Yes!  But, for this case it's quite unique -- I'm just trying to cover the inside of the button with an overlay, not so much a shadow.  This "shadow" is unlikely to be reused.